### ISSUE TITLE
Description list syntax highlighting

### ISSUE THREAD

**Issue by Mogztter** [asciidoctor/asciidoctor-intellij-plugin#472]
The syntax highlighting (parser) for a description list is wrong if the term contains `::`. It does not happen often but here's a concrete example when describing Ruby modules/classes:

```
Asciidoctor:: `Asciidoctor` module
Asciidoctor::AbstractNode:: An `AbstractNode`
```

![dlist](https://user-images.githubusercontent.com/333276/80344023-2dd27e00-8867-11ea-985e-401d908a3c9a.png)

As you can see `AbstractNode::` is not highlighted correctly.

**AsciiDoc Plugin Version**
```
0.30.67
```

**IDE Name and Version**
```
WebStorm 2020.1
Build #WS-201.6668.106, built on April 6, 2020
Runtime version: 11.0.6+8-b765.25 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Linux 4.15.0-96-generic
GC: ParNew, ConcurrentMarkSweep
Memory: 1931M
Cores: 8
Registry: debugger.watches.in.variables=false
Non-Bundled Plugins: org.jetbrains.plugins.slim, net.seesharpsoft.intellij.plugins.csv, BashSupport, org.asciidoctor.intellij.asciidoc, org.toml.lang
Current Desktop: XFCE
```

**Comment by ahus1** [comment]
Well spotted! It's fixed and will be part of the next release.

**Comment by ahus1** [comment]
This has been fixed in 0.30.68 that is now available as a pre-release. That pre-release contains a larger refactoring on parsing/lexing table cells. Please have an eye on regressions in that area.
